### PR TITLE
 Update AMF SDK To 1.4.18  And Fix Escape Character at zh_TW

### PR DESCRIPTION
**Pull request by ColaLiuLiu**
### Description

Update AMF SDK To 1.4.18
Replace PrePassMode PropertyInfo 

Fix Escape Character at zh_TW

### Motivation and Context
Update AMF SDK and update Prepass Mode Code on h264

Fix Lang/zh_TW  words show on OBS

### How Has This Been Tested?
This Test with Compiler to OBS and Test Streaming on Twitch
with Windows 1903 / OBS Setting 709 NV12 partial Color Space

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [x] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

File: resources/locale/zh-TW.ini [hunk 1]
```diff
@@ -28,7 +28,7 @@ CodingType.Description="編碼最終封包時所使用的編碼類型。\n- 「\
 MaximumReferenceFrames="最大參考訊框"
 MaximumReferenceFrames.Description="輸出檔案的最大參考訊框。這對資源這個功能的編碼器的畫質有絕對的影響。\n\n這個選項是靜態的，且不能在編碼時變更。"
 RateControlMethod="速率控制方法"
-RateControlMethod.Description="應使用何種方法以控制（位元）速率？\n- 「\@RateControlMethod.CQP\@」為每個訊框分配固定量化參數，建議用於高品質至接近無損的錄製。\n-「\@RateControlMethod.CBR\@」嘗試盡可能地接近 \@Bitrate.Target\@，可以選擇用 \@FillerData\@ 填充，建議用於串流。\n- 「\@RateControlMethod.VBR\@」嘗試盡可能地接近 \@Bitrate.Peak\@，若可能，則盡可能地降低至 \@Bitrate.Target\@，建議用於小尺寸錄製。\n- 「\@RateControlMethod.VBRLAT\@」與「@RateControlMethod.VBR\@」類似，但將編碼延遲列入考量。"
+RateControlMethod.Description="應使用何種方法以控制（位元）速率？\n- 「\@RateControlMethod.CQP\@」為每個訊框分配固定量化參數，建議用於高品質至接近無損的錄製。\n-「\@RateControlMethod.CBR\@」嘗試盡可能地接近 \@Bitrate.Target\@，可以選擇用 \@FillerData\@ 填充，建議用於串流。\n- 「\@RateControlMethod.VBR\@」嘗試盡可能地接近 \@Bitrate.Peak\@，若可能，則盡可能地降低至 \@Bitrate.Target\@，建議用於小尺寸錄製。\n- 「\@RateControlMethod.VBRLAT\@」與「\@RateControlMethod.VBR\@」類似，但將編碼延遲列入考量。"
```
Comment (ghost) [inline resources/locale/zh-TW.ini:31]: This change is unrelated and should also be handled on Crowdin (https://crowdin.com/project/obs-studio). I've fixed this particular issue, so the next translation update fixes the escape character.

**Comment by RytoEX** [comment]
Closing this PR as this encoder has been deprecated in favor of a new implementation in the upcoming OBS Studio 28 release, which currently uses AMF 1.4.24.
